API/Controllers/AccountController.cs

```diff
@@ -37,6 +37,8 @@ namespace API.Controllers;
 /// </summary>
 public class AccountController : BaseApiController
 {
+    private static readonly string BadCredentialsMessage = "Your credentials are not correct";
```

Comment:
Can you add a comment here referencing the issue and mention avoiding localization which can cause multiple enumeration.